README.md

```diff
@@ -41,14 +45,14 @@ To support [OpenStack Style Guidelines](http://google-styleguide.googlecode.com/
   ```
 
 ### Settings
-You can configure linter-flake8 as nearly all [Atom](https://atom.io/) modules by editing plugin settings in *Atom->Preferences->Packages->linter-flake-8*.
+You can configure linter-flake8 as nearly all [Atom](https://atom.io/) modules by editing the package Settings in *Atom -> Preferences -> Packages -> linter-flake8*.
 
 ![img](https://cloud.githubusercontent.com/assets/4278113/8768488/91fffae2-2e3f-11e5-9783-9513b882cbc0.png)
 
-Or if you prefere you can use Atom *config.cson* file *~/.atom/config.cson* (click *Open Your Config*
+Or if you prefer you can use Atom *config.cson* file *~/.atom/config.cson* (click *Open Your Config*
 in *Atom* menu).
 
-In configuration you can specify executable directory if node hasn't it in **$PATH**. Example:
+In the Settings panel you can specify executable directory if node hasn't it in **$PATH**. Example:
```

Comment:
"...if Atom isn't able to determine it automatically. Example:" instead?

(At the very least "hasn't it in" needs to be re-written)